tests/core/evaluation/test_marker_tracker_loader.py

```diff
@@ -0,0 +1,85 @@
+from rasa.shared.core.events import UserUttered
+from rasa.shared.core.trackers import DialogueStateTracker
+from rasa.shared.core.domain import Domain
+from rasa.shared.exceptions import RasaException
+from rasa.core.evaluation.marker_tracker_loader import MarkerTrackerLoader
+from rasa.core.tracker_store import InMemoryTrackerStore, TrackerStore
+import pytest
+
+
+@pytest.fixture
+def marker_trackerstore() -> TrackerStore:
+    """Set up a TrackerStore with 5 trackers in it."""
+    domain = Domain.empty()
+    store = InMemoryTrackerStore(domain)
+    for i in range(5):
+        tracker = DialogueStateTracker(str(i), None)
+        tracker.update_with_events([UserUttered(str(j)) for j in range(10)], domain)
+        store.save(tracker)
+
+    return store
+
+
+def test_load_sample(marker_trackerstore: TrackerStore):
+    """Test loading trackers using 'sample' strategy."""
+    loader = MarkerTrackerLoader(marker_trackerstore, "sample", 3)
+    result = list(loader.load())
+
+    assert len(result) == 3
+
+    for item in result:
+        assert marker_trackerstore.exists(item.sender_id)
+
+
+def test_load_first_n(marker_trackerstore: TrackerStore):
+    """Test loading trackers using 'first_n' strategy."""
+    loader = MarkerTrackerLoader(marker_trackerstore, "first_n", 3)
+    result = list(loader.load())
+
+    assert len(result) == 3
+
+    for item in result:
+        assert marker_trackerstore.exists(item.sender_id)
+
+
+def test_load_all(marker_trackerstore: TrackerStore):
+    """Test loading trackers using 'all' strategy."""
+    loader = MarkerTrackerLoader(marker_trackerstore, "all")
+    result = list(loader.load())
+
+    assert len(result) == len(marker_trackerstore.keys())
```

Comment:
Pycharm is complaining about `marker_trackerstore.keys()`: `Expected type 'Sized', got 'Iterable[str]' instead`. Any way to address this warning?